hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -4078,38 +4103,29 @@ func (r *HostedClusterReconciler) validateReleaseImage(ctx context.Context, hc *
 		currentVersion = &version
 	}
 
-	// Validate release image is multi-arch
-	if hc.Spec.Platform.Type == hyperv1.AWSPlatform && hc.Spec.Platform.AWS.MultiArch {
-		isMultiArchReleaseImage, err := registryclient.IsMultiArchManifestList(ctx, hc.Spec.Release.Image, pullSecretBytes)
-		if err != nil {
-			return fmt.Errorf("failed to determine if release image multi-arch: %w", err)
-		}
-
-		if !isMultiArchReleaseImage {
-			return fmt.Errorf("release image is not a multi-arch image")
-		}
-	}
-
-	// Validate each NodePool CPU arch matches the management cluster's CPU arch when the Hosted Cluster is not multi-arch
-	if hc.Spec.Platform.Type == hyperv1.AWSPlatform && !hc.Spec.Platform.AWS.MultiArch {
-		nodePoolList := &hyperv1.NodePoolList{}
-		if err := r.List(ctx, nodePoolList, client.InNamespace(hc.Namespace)); err != nil {
-			return fmt.Errorf("failed to get list of nodepools for cpu arch validation: %w", err)
-		}
-
-		mgmtClusterCPUArch := runtime.GOARCH
+	minSupportedVersion := supportedversion.GetMinSupportedVersion(hc)
 
-		for _, nodePool := range nodePoolList.Items {
-			if err := hyperutil.DoesMgmtClusterAndNodePoolCPUArchMatch(mgmtClusterCPUArch, nodePool.Spec.Arch); err != nil {
-				return err
-			}
-		}
+	return supportedversion.IsValidReleaseVersion(&version, currentVersion, &supportedversion.LatestSupportedVersion, &minSupportedVersion, hc.Spec.Networking.NetworkType, hc.Spec.Platform.Type)
+}
 
+// isMultiArchNPCapable returns true if the HostedCluster release image has a multi-arch payload and false if it is a
```

Comment:
based on what it does shouldn't be just named isMultiArchHC? or even isMultiArch?

Reply:
I have no problem renaming this. It's now `isMultiArchHC `.